test/api/Auth.js

```diff
@@ -226,6 +196,140 @@ describe('Auth', () => {
         await User.deleteOne({email: user.email});
       }
     });
+
+    it('Should create an audit log entry on successful signup', async () => {
+      const registerPayload = {
+        email: 'newuser@example.com',
+        password: 'Passw0rd123!',
+        firstName: 'Testfirst',
+        lastName: 'Testlast'
+      };
+
+      // ensure Audit log and User DB starts fresh before this test
+      await AuditLog.deleteMany({});
+      await User.deleteOne({email: registerPayload.email});
+
+      const res = await test.sendPostRequest('/api/Auth/register', registerPayload);
+      expect(res).to.have.status(OK);
+
+      const auditEntry = await AuditLog.findOne({
+        action: AuditLogActions.SIGN_UP,
+        'details.email': registerPayload.email
+      }).lean();
+
+      expect(auditEntry).to.exist;
+      expect(auditEntry).to.have.property('userId');
+      expect(auditEntry.details).to.have.property('email', registerPayload.email);
+
+    });
+  });
+
+  describe('/POST login', () => {
```

Comment:
i thnk we can move the tests into the below section, we already have it for /POST login

https://github.com/SCE-Development/Clark/blob/5b59f83035cfd1f1e508739b0e0ffff861c21caa/test/api/Auth.js#L146